### PR TITLE
Create peters-test.html

### PR DESCRIPTION
**Pull request by petr0n**
Please review

### CODE REVIEW

**Comment by DevEngberg** [comment]
Looks Great!

File: peters-test.html [hunk 1]
```diff
@@ -0,0 +1 @@
+html goes here
```
Comment (nzhengs) [inline peters-test.html]: This is an invalid html code

Comment (petr0n) [reply peters-test.html]: Ok I'll fix

**Comment by petr0n** [comment]
How's this update?

**Pull request merged**